video/out/opengl/lcms.c

```diff
@@ -184,12 +192,28 @@ bool gl_lcms_set_memory_profile(struct gl_lcms *p, bstr profile)
     return true;
 }
 
+// Guards against NULL and uses bstr_equals to short-circuit some special cases
+static bool vid_profile_eq(struct AVBufferRef *a, struct AVBufferRef *b)
+{
+    if (a == b)
+        return true;
```

Comment:
@wm4 Not true: `(struct bstr){ a->data, a->size }` could be derefencing a NULL pointer, so we still need to manually check the cases where one is NULL and the other is not (or both are NULL, in which case they're equal again).